chef/cookbooks/crowbar/templates/default/crowbar-lbaas-evacuation.sh.erb

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+#
+# This script migrates all neutron lbaasv2 loadbalancers from the node
+# that is passed in via the command line argument.
+
+LOGFILE=/var/log/crowbar/node-upgrade.log
+UPGRADEDIR=/var/lib/crowbar/upgrade
+mkdir -p "`dirname "$LOGFILE"`"
+exec >>"$LOGFILE" 2>&1
+
+hostname=$1
+if [[ "$2" = "delete-ns" ]]; then
+    delete_ns="--delete_namespaces --nosource_agent_restart"
+else
+    delete_ns=""
+fi
+
+log()
+{
+    set +x
+    echo "[$(date --iso-8601=ns)] [$$] $@"
+    set -x
+}
+
+log "Executing $BASH_SOURCE"
+
+set -x
+
+mkdir -p $UPGRADEDIR
+rm -f $UPGRADEDIR/crowbar-lbaas-evacuation-failed
+
+systemctl --quiet is-active  openstack-neutron-lbaasv2-agent.service
+ret=$?
+if [ $ret = 0 ] ; then
+    use_crm=<%= @use_ha ? "--use_crm" : "" %>
```

Comment:
Actually, in our case we will always call the tool with ```--use_crm``` there is no non-ha upgrade scenario that will call this script AFAICs. I just added that for completeness. I really didn't want to go into the mess of parsing crm output here though, when we have the ```use_ha``` template variable already in the recipe.